src/lib/y2partitioner/format_mount/options.rb

```diff
@@ -23,47 +23,6 @@
 require "y2storage"
 
 module Y2Storage
-  # FIXME: Monkey patch of Y2Storage::PartitionId it should moved to
```

Comment:
WTF! How am I supposed to test and use this? Please wake up and do the right thing.